typhos/tests/test_positioner.py

```diff
@@ -274,3 +274,30 @@ def has_limit_error():
         assert 'LimitError' in widget.ui.status_label.text()
 
     qtbot.waitUntil(has_limit_error, timeout=1000)
+
+
+@pytest.mark.no_gc
+def test_positioner_widget_long_status_text(motor_widget, qtbot):
```

Comment:
I had to remind myselfl aoub this `no_gc` call ([from an earlier CI overhaul](https://github.com/pcdshub/typhos/pull/607#issuecomment-2145677518)), but its usage here jives with my recollection of when and where we were applying this

Reply:
Do we need to add qtbot as a fixture argument to this test?  We give qtbot the motor widget in fixture construction, and it's not used to `waitUntil` or similar in the test body

Reply:
I think you're right, this is probably a copy/paste test case no thinking situation